mrmustard/math/backend_tensorflow.py

```diff
@@ -245,6 +245,9 @@ def maximum(self, a: tf.Tensor, b: tf.Tensor) -> tf.Tensor:
     def minimum(self, a: tf.Tensor, b: tf.Tensor) -> tf.Tensor:
         return tf.minimum(a, b)
 
+    def moveaxis(self, a: tf.Tensor, old: tf.Tensor, new: tf.Tensor) -> tf.Tensor:
```

Comment:
I believe the type hint for old and new would still be `Union[int, Sequence[int]]` or `int | Sequence[int]`

Reply:
In fact all of them could just be `a: Tensor, old: int | Sequence[int], new: int | Sequence[int]`